c7n/resources/ecs.py

```diff
@@ -947,7 +946,7 @@ def process_cluster_resources(self, client, cluster_id, container_instances):
             r = client.describe_container_instances(
                 cluster=cluster_id,
                 include=['TAGS'],
-                containerInstances=container_instances).get('containerInstances', [])
+                containerInstances=service_set).get('containerInstances', [])
```

Comment:
This change could do with some explanation. It doesn't match the other cleanup pattern.

Reply:
Right, this was actually somewhat a bug, since the loop was getting the whole list of instances every time,rather than the slice it was supposed to (which is the reason of the looping).
This didn't really break anything, it was just doing extra works.